supportedBackends/google/pipelines/batch/src/main/scala/cromwell/backend/google/pipelines/batch/GcpBatchJob.scala

```diff
@@ -174,55 +174,49 @@ final case class GcpBatchJob (
       .build
   }
 
-  def submitJob(): Unit = {
-
-    try {
```

Comment:
The changes are mostly indentation because I removed the `try/catch` block to propagate the errors